test/dshell/sameenv.d

```diff
@@ -8,8 +8,8 @@ void main()
     run("$OUTPUT_BASE/printenv$EXE", File(envFromExe, "wb"));
     run("$DMD -m$MODEL -run $EXTRA_FILES/printenv.d", File(envFromRun, "wb"));
 
-    const fromExe = readText(envFromExe);
-    const fromRun = readText(envFromRun);
+    const fromExe = readOutput(envFromExe);
```

Comment:
The exe output won't have the debug message to filter out.

Reply:
The output still needs to be stripped from \r otherwise the output produces duplicate newlines.